src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -803,10 +803,13 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
         
 
         public FormValidation doCheckAdminlist(@QueryParameter String value) throws ServletException {
-            if (!adminlistPattern.matcher(value).matches()) {
-                return FormValidation.error("GitHub username may only contain alphanumeric characters or dashes "
-                        + "and cannot begin with a dash. Separate them with whitespaces.");
+            for (String admin : value.split("\\s")) {
```

Comment:
Right, I updated the PR to split on any space character. That should do the trick.

Apologies for the very long delay here.